doc/examples/_valid_examples.toml

```
@@ -1,14 +1,26 @@
 [main]
 readme = """
+:html_theme.sidebar_secondary.remove:
+
 .. _examples:
 
+
 Examples
 ========
 
-.. toctree::
-   :maxdepth: 1
+.. note::
+    The examples here are some uses of the analysis and visualization functionality
+    of DIPY, with example data from actual neuroscience experiments, or with
+    synthetic data, which is generated as part of the example.
+
+    All the examples presented in the documentation are generated from
+    *fully functioning* python scripts, which are available as part of the
+    source distribution in the `doc/examples` folder.
+
+    If you want to replicate a particular analysis or visualization, simply copy
+    or download the relevant `.py` script from the source distribution, edit out
```

I'd remove "the source distribution", because some users might not know what that means exactly. And the '.py' files are in the website on each page as well, so probably easier to get them from there?

```suggestion
    or download the relevant `.py` script, edit out
```